node-export-server/test/test_overpass.osm

```diff
@@ -0,0 +1,113 @@
+﻿<?xml version="1.0" encoding="UTF-8"?><osm version="0.6" generator="Overpass API 0.7.55.7 8b86ff77">
```

Comment:
if we want to be really diligent, maybe we would try to read this generator tag?

Reply:
We could, but since we pass the datasource as a path param I think we can stick with current implementation